### PR TITLE
Dashboard/projects translations

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: src/projects/service.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import Vue from 'vue';
 
 const ProjectsService = {
-  list: params => Vue.http.get(`${Vue.config.projectsUrlBase}/api/v1/en/projects`, { params }),
+  list: (language, params) => {
+    // Note : this is a patch due to /projects non-consistent language
+    // eslint-disable-next-line no-unused-vars
```
Comment (DanielBrierton) [inline src/projects/service.js]: This doesn't seem necessary..?

Comment (Wardormeur) [reply src/projects/service.js]: Well, the linter was complaining about lang not being used :/

**Comment by Wardormeur** [comment]
Aaaand linter failed. My bad, will fix asap

File: src/projects/service.js (hunk 1 above)
Comment (DanielBrierton) [reply src/projects/service.js]: But, like... It is used. Also, conflicts! 😄

Comment (Wardormeur) [reply src/projects/service.js]: Will have another look at it while solving the conflict, but pretty sure I had the same reaction than you did

**Comment by DanielBrierton** [comment]
Cypress tests seem a little flakey... 😕

**Pull request merged**